README.md

```diff
@@ -1,29 +1,48 @@
+# Chrony Charm
+
 [![CharmHub Badge](https://charmhub.io/chrony/badge.svg)](https://charmhub.io/chrony)
 [![Publish to edge](https://github.com/canonical/chrony-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/chrony-operator/actions/workflows/publish_charm.yaml)
 [![Promote charm](https://github.com/canonical/chrony-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/chrony-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
 for deploying and managing the [Chrony](https://chrony-project.org) NTP server 
 in your systems.
 
 This charm simplifies the configuration and maintenance of `chrony` across a 
 range of environments, enabling basic time provision and synchronization with 
 NTP servers.
 
-## Project and community
+## Get started
+In this section, we will deploy the Chrony charm, which can be used as a time source for NTP clients.  
+You’ll need a workstation, e.g., a laptop, with sufficient resources to launch a virtual machine with 4 CPUs, 8 GB RAM, and 50 GB disk space.
+
+### Set up
+You can follow the tutorial [here](https://juju.is/docs/juju/set-up--tear-down-your-test-environment#heading--set-up-automatically) to set up a test environment for Juju with LXD.
+
+### Deploy
+Deploy the Chrony charm using the `juju deploy` command.
```

Comment:
If the purpose of including the prompt mark was to show the user that they are within the VM, then I suggest modifying this instruction to indicate that. Something like, "From inside the virtual machine, deploy the Chrony charm using the `juju deploy` command."